### PR TITLE
Dynamically add a credit to LHI in the footer

### PR DESCRIPTION
**Pull request by sraby**
This PR credits LHI for helping with the implementation of EHP. It was requested that we credit them via a bit of text in the footer on all EHP pages.

### CODE REVIEW

File: frontend/lib/ui/footer.tsx [hunk 1]
```diff
@@ -1,42 +1,55 @@
...
-              legal advice and must not be used as a substitute for the advice
-              of a lawyer qualified to give advice on legal issues pertaining to
-              housing. We can help direct you to free legal services if
-              necessary.
+              Made with NYC ♥ by the team at{" "}
+              <a href="https://justfix.nyc">JustFix.nyc</a>
             </p>
-            <p>JustFix.nyc is a registered 501(c)(3) nonprofit organization.</p>
+            {/* Include credit for LHI on all ehp routes */}
+            {props.pathname && props.pathname.includes("/ehp") && (
```
Comment (sraby) [inline frontend/lib/ui/footer.tsx:29]: I wonder, is there a better way to target all routes with the "ehp" prefix?

Comment (toolness) [reply frontend/lib/ui/footer.tsx:29]: Oh yeah, you should use `props.pathname.startsWith(Routes.locale.ehp[ROUTE_PREFIX])`!  If this kind of thing becomes something we do a lot we can make a new function to make it easier, e.g. `isPathnameInside(pathname, Routes.locale.ehp)`.

Comment (toolness) [reply frontend/lib/ui/footer.tsx:29]: Hmm so CodeClimate was complaining that this function is now 42 lines and I squelched it, but now that I'm reading the code I wonder if it makes sense to factor out the LHI mention (including the conditional logic) into a separate component?  Eh, up to you.

Comment (sraby) [reply frontend/lib/ui/footer.tsx:29]: Ahhh yeah... I thought about it but thought that the added complexity of having to pass the pathname prop down wasn't worth it?

Comment (sraby) [reply frontend/lib/ui/footer.tsx:29]: On the second thought, it would look a bit cleaner. Ok let me add that.

**Pull request merged**